tools/uniswapv2/detect.py

```diff
@@ -65,7 +64,7 @@ def pairs_from_ethereum(ethereum: EthereumManager) -> Dict[str, Any]:
     for idx, chunk in enumerate(chunks):
         print(f'Querying univ2 pairs chunk {idx + 1} / {len(chunks)}')
         result = multicall_specific(ethereum, univ2factory, 'allPairs', chunk)
-        pairs.extend([to_checksum_address(x[0]) for x in result])
+        pairs.extend([deserialize_ethereum_address(x[0]) for x in result])
```

Comment:
handle a potential `DeserializationError` here. No fancy way needed. Just a `print()` of the error followed by a `sys.exit(1)`.

This is just a script to detect uniswap lp pools.